README.rst

```diff
@@ -87,7 +85,8 @@ GitHub`_. We appreciate contributions of any kind.
 .. _Developer documentation: DEVELOP.rst
 .. _managed on GitHub: https://github.com/crate/crate-python
 .. _PyPI: https://pypi.org/
-.. _SQLAlchemy: https://www.sqlalchemy.org
-.. _SQLAlchemy dialect: https://docs.sqlalchemy.org/dialects/
+.. _SQLAlchemy: https://www.sqlalchemy.org/
+.. _sqlalchemy-cratedb: https://github.com/crate/sqlalchemy-cratedb
+.. _sqlalchemy-cratedb documentation: https://cratedb.com/docs/sqlalchemy-cratedb/
 .. _StackOverflow: https://stackoverflow.com/tags/cratedb
 .. _support channels: https://crate.io/support/
```

Comment:
Minor and unrelated; maybe we can upgrade to `cratedb.com`

Reply:
Improved with 7dfff631704.